src/machine/behavior/is/other/put.rs

```diff
@@ -1,6 +1,18 @@
 use machine::state::State;
+use machine::state::sr::R;
 
-pub fn put(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn put(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+
+    // Execute
+    if x > 31 {
+        panic!("no register");
+    }
```

Comment:
Putting is not allowed for any special register. See the table on page 21 for details.